airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/CopyDestination.java

```diff
@@ -71,7 +71,7 @@ public AirbyteConnectionStatus check(final JsonNode config) {
       AbstractJdbcDestination.attemptSQLCreateAndDropTableOperations(outputSchema, database, nameTransformer, getSqlOperations());
 
       return new AirbyteConnectionStatus().withStatus(AirbyteConnectionStatus.Status.SUCCEEDED);
-    } catch (final ConnectionErrorException ex) {
+    } catch (final ConfigErrorException ex) {
       LOGGER.info("Exception while checking connection: ", ex);
       final String message = getErrorMessage(ex.getStateCode(), ex.getErrorCode(), ex.getExceptionMessage(), ex);
```

Comment:
Same thing here, would want to remove this line and supplement message with `ex.getDisplayMessage()` instead